src/utils/hooks/useBoolean.ts

```diff
@@ -0,0 +1,25 @@
+import { useState, useCallback } from 'react';
+
+export default function useBoolean(initialValue: boolean) {
+  const [value, setValue] = useState(initialValue);
+  const setValueTrue = useCallback(
```

Comment:
반환하는 값들에 비해 useBoolean 이란 네이밍이 어색한듯 합니다 😅 
`useBooleanState` 는 어떨까요?
```suggestion
export default function useBooleanState(initialValue: boolean) {
  const [value, setValue] = useState(initialValue);
  const setValueTrue = useCallback(
```